doc/dev/crimson/crimson.rst

```diff
@@ -175,6 +227,10 @@ The following options can be used with ``vstart.sh``.
 ``--bluestore``
     Use the alienized BlueStore as the object store backend. This is the default.
 
+``--crimson-smp``
+    Bluestore only. number of cores to use for each OSD. The rest of the available
```

Comment:
BlueStore only.  The number of cores to use for each OSD.  The balance of available cores (as determined by `nproc`) will be assigned to BlueStore.